designsafe/urls.py

```diff
@@ -138,10 +138,14 @@
     # version check
     url(r'^version/', des_version),
 
+    # old NEES urls
+    url(r'^warehouse/project/([0-9]+)/?', redirect_old_nees),
+    url(r'^warehouse/experiment/([0-9]+)/project/([0-9]+)/?', redirect_old_nees),
+    url(r'^warehouse/hybrid/([0-9]+)/project/([0-9]+)/?', redirect_old_nees),
```

Comment:
If you only need the last number, you can opt to capture only that value from the url
```suggestion
    url(r'^warehouse/hybrid/[0-9]+/project/([0-9]+)/?', redirect_old_nees),
```